pkg/api/user.go

```diff
@@ -237,7 +237,7 @@ func (hs *HTTPServer) handleUpdateUser(ctx context.Context, cmd user.UpdateUserC
 		return response.ErrOrFallback(http.StatusInternalServerError, "Failed to update user", err)
 	}
 
-	return response.Success("User updated")
+	return response.Success("")
```

Comment:
This is to avoid having this message pop-up alongside achievements when updating user object